holocron_api/api/models/standard.py

```diff
@@ -4,12 +4,26 @@
 from Ad_Network import Ad_Network
 
 
+class Program(models.Model):
+    program_key = models.CharField(max_length=100, primary_key=True)
+    program_name = models.CharField(max_length=100)
+    program_description = models.CharField(max_length=140)
+    created_by = models.CharField(max_length=100)
+    program_notes = models.CharField(max_length=140, blank=True)
+    start_date = models.DateField('Start Date', blank=True, null=True)
+    end_date = models.DateField('End Date', blank=True, null=True)
+    created_on = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.program_key
+
+
 class Campaign(models.Model):
-    campaign_id = models.AutoField(primary_key=True)
-    campaign_key = models.CharField(max_length=100, blank=True, null=True,
-                                    unique=True)
+    campaign_key = models.CharField(max_length=100, primary_key=True)
```

Comment:
I think Campaign (The "New" one) should have an "id" not a key. There's probably little value in making someone come up with a "key"... an auto-number ID is probably better.